### PR TITLE
Release 2.1.1

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +29,7 @@
 
 **Merged pull requests:**
 
+- Release 2.1.0 [\#118](https://github.com/voxpupuli/puppet_webhook/pull/118) ([dhollinger](https://github.com/dhollinger))
```
Comment (ekohl) [inline CHANGELOG.md:32]: We typically skip these. Is it intentional that these are all visible again?

File: Rakefile [hunk 2]
```diff
@@ -40,6 +40,7 @@ if ENV['SINATRA_ENV'] != 'production'
     config.user = 'voxpupuli'
     config.project = 'puppet_webhook'
     config.future_release = PuppetWebhook::VERSION
+    config.exclude_labels = %w{duplicate.question.invalid.wontfix.wont-fix.modulesync.skip-changelog}
```
Comment (ekohl) [inline Rakefile:43]: I think the dots break things

File: CHANGELOG.md (hunk 1 above)
Comment (dhollinger) [reply CHANGELOG.md:32]: Nope. I'm working on that.

Comment (dhollinger) [reply CHANGELOG.md]: @ekohl Should be fixed now. Looks like it also cleaned up some other issues that shouldn't have been in the changelog

File: Rakefile (hunk 2 above)
Comment (dhollinger) [reply Rakefile]: Yup, fixed. That's what I get for copy/paste from one nvim window to another, instead of yank/paste

**Pull request merged**